tests/text_reparser/plugins/contact_admin_info_test.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+require_once __DIR__ . '/../../../phpBB/includes/functions.php';
+require_once __DIR__ . '/../../../phpBB/includes/functions_content.php';
+require_once __DIR__ . '/../../test_framework/phpbb_database_test_case.php';
```

Comment:
They're not when the test is run as part of the test suite. However, they're needed to run the test separately. If I remove them, the next time there's a bug related to the tested class the choice will be between waiting 20+ minutes for the test suite to run or spending X amount of time figuring out which files are required to run the test alone.

I'll remove them if you still think that's worth it.